packages/koehnlab/spin_hamiltonians/g_tensor.py

```diff
@@ -0,0 +1,72 @@
+import numpy as np
+from .phys_const import muBcm,g_e
+
+
+def get_magnetic_moment_matrix(s,l,unit: bool):
+        """
+        Computes the magnetic moment of the system in the basis of given spin and angular momentum matrix
+        Procedure taken from: L. F. Chibotaru, L. Ungur; Ab initio calculation of anisotropic magnetic properties of complexes. I. Unique definition of
+        pseudospin Hamiltonians and their derivation. J. Chem. Phys. 14 August 2012; 137 (6): 064112. https://doi.org/10.1063/1.4739763
+        Args:
+        ---------------------
+        s -- spin matrix in Js
+        l -- angular momentum matrix in (kg*m^2)/s
+        unit -- Boolean if in multiples of bohr magneton
+
+        Returns:
+        ---------------------
+        muMat -- matrix of magnetic moment in J/T
```

Comment:
```suggestion
        muMat -- matrix of magnetic moment
```
careful with those unit specifications. We almost never get SI units. Besides, the units of the computed results depend entirely on the units of the inputs.